apps/components/WordOccurrencesWrapper.js

```diff
@@ -49,7 +61,7 @@ const RelevantWordInputRow = ( props ) => {
 		props.onChange( event.target.value, "_stem", props.index );
 	}
 	function changeOccurrences( event ) {
-		props.onChange( event.target.value, "_occurrences", props.index );
+		props.onChange( parseInt( event.target.value ), "_occurrences", props.index );
```

Comment:
Please ass the `radix` here as well. This makes sure that we will always convert the `event.target.value` to a decimal. Otherwise, it depends on the start of the string (which is something that we don't want).

```suggestion
		props.onChange( parseInt( event.target.value, 10 ), "_occurrences", props.index );
```